templates/output.html.twig

```diff
@@ -23,6 +23,12 @@
                         {{ msg('engine-name-tesseract') }}
                     </label>
                 </div>
+                <div class="radio">
+                    <label>
+                        <input type="radio" name="engine" value="transkribus" {% if engine == 'transkribus' %}checked{% endif %} />
+                       Transkribus OCR
```

Comment:
Needs internationalization (i.e. using `msg()` and adding to `en.json` and `qqq.json`). Copy how the others are handled.